plugins/modules/swim_workflow_manager.py

```diff
@@ -1225,15 +1225,27 @@ def get_diff_tagging(self):
             )
             self.log("Received API response from 'remove_golden_tag_for_image': {0}".format(str(response)), "DEBUG")
 
-        if response:
-            task_details = {}
-            task_id = response.get("response").get("taskId")
+        if not response:
+            self.status = "failed"
+            self.msg = "Did not get the response of API so cannot check the Golden tagging status of image - {0}".format(image_name)
```

Comment:
Can we rephrase ?
self.msg = "Failed to get response from API 'API_name': Unable to verify Golden tag status for image '{0}'.".format(image_name)

(OR)
self.msg = "API 'Name' response unavailable: Unable to verify Golden tag status for image '{0}'.".format(image_name)